subprojects/provider/src/main/kotlin/org/gradle/kotlin/dsl/GroovyInteroperability.kt

```diff
@@ -121,6 +121,30 @@ class KotlinClosure2<in T : Any?, in U : Any?, V : Any>(
 }
 
 
+/**
+ * Adapts a trivariate Kotlin function to a binary Groovy [Closure].
```

Comment:
:o: I would use _ternary_ here instead of _trivariate_ since we are using `unary` and `binary` for the other types.

Reply:
❓You mean `ternary` Groovy closure, right?